chart/templates/flower/flower-service.yaml

```diff
@@ -33,6 +33,10 @@ metadata:
 {{- with .Values.labels }}
 {{ toYaml . | indent 4 }}
 {{- end }}
+{{- with .Values.flower.service.annotations }}
+  annotations:
```

Comment:
Can you add a simple test in https://github.com/apache/airflow/blob/master/chart/tests/test_flower.py please

Reply:
done